src/test/unit/services/country.spec.js

```
@@ -0,0 +1,56 @@
+const countryService = require('~/services/countries');
```

ESLint

[error] Extra semicolon.

(semi)